doc/nrf/libraries/bluetooth_services/services/hrs_client.rst

```diff
@@ -54,6 +46,14 @@ The following |NCS| samples use this library:
 * :ref:`central_and_peripheral_hrs`
 * :ref:`bluetooth_central_hr_coded`
 
+Additional information
+**********************
+
+.. note::
+   Do not use any of the values in the :c:struct:`bt_hrs_client` object structure directly.
+   All values that should be accessed have accessory functions.
+   The structure is fully defined because the application must be able to allocate memory for it.
+
 Dependencies
 ************
 
```

Comment:
Line 63 and 64:
 * :file:\`include/bluetooth/gatt.h\`
 * :file:\`include/bluetooth/conn.h\`